addons/stock_dropshipping/data/stock_dropshipping_demo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="product_mug" model="product.product">
+        <field name="name">Mug</field>
+        <field name="detailed_type">product</field>
+        <field name="list_price">15.0</field>
+        <field name="route_ids" eval="[(4, ref('route_drop_shipping'))]"></field>
+    </record>
+
+    <record id="product_supplierinfo_mug" model="product.supplierinfo">
```

Comment:
Ok for the Sales order but please use an existing product with a supplier info

Reply:
Indeed, I made the change. Thanks for the review.